grunt/protractor_coverage.js

```diff
@@ -1,14 +1,17 @@
+"use strict";
+
 module.exports = {
     options: {
         keepAlive: false,
         noColor: false,
         coverageDir: "coverage/protractor",
         args: {
-            baseUrl: "http://localhost:3000"
+            baseUrl: "http://localhost:9000"
         }
     },
     local: {
         options: {
+            keepAlive: true,
```

Comment:
We discussed including bail-fast locally, I've not included it yet in the dominatr changes though.